packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -0,0 +1,25 @@
+module KLDivergence = {
+  let logFn = Js.Math.log
```

Comment:
Probably good to declare the base, or write it in a comment. I assume this is base e but am not 100% sure.